tools/testing-server/constants.js

```diff
@@ -59,6 +59,7 @@ module.exports.defaultInitBlock = {
   metrics: { enabled, autoStart },
   obfuscate: undefined,
   page_action: { enabled },
+  user_actions: { enabled: false },
```

Comment:
Does this still need to happen with the adjustments that have been made to the page actions and user actions tests?  I think we'd be better off if the default state of the agent matched the default state of the testing server configs, if feasible

Reply:
I didn't fix every insightCapture case that's why. I'm just going to throw in a separate story into backlog for that.

Reply:
That works for me